docs/getting-started-with-dynamodb.md

```diff
@@ -0,0 +1,34 @@
+# Getting Started with Scalar DB on Dynamo DB
+
+## Overview
+This document briefly explains how you can get started with Scalar DB on Dynamo DB with a simple electronic money application.
```

Comment:
```suggestion
This document briefly explains how you can get started with Scalar DB on DynamoDB with a simple electronic money application.
```

Reply:
Thank you for your suggestions.
Document updated accordingly.